tests/unit/multivariate/test_gaussian.py

```diff
@@ -423,7 +423,7 @@ def test_sample_random_state(self):
         result = instance.sample(5)
 
         # Check
-        pd.testing.assert_frame_equal(result, expected_result, check_less_precise=True)
```

Comment:
did we need the `check_less_precise=True`?

Reply:
This is now deprecated, but I can do `check_exact=True` if we need to ensure exact numbers

Reply:
Actually they suggest using `rtol` 🤔

Reply:
I don't think we were trying to check exact before but actually the opposite. The parameter was giving some tolerance and I'm wondering if not doing it anymore will lead to a fragile test. Here is another possible replacement https://github.com/pandas-dev/pandas/pull/30562

Reply:
But if the test passes without any issues we can just get rid of it